beacon-chain/rpc/prysm/v1alpha1/validator/proposer_execution_payload.go

```diff
@@ -299,7 +289,7 @@ func getParentBlockHashPostMerge(st state.BeaconState) ([]byte, error) {
 	if err != nil {
 		return nil, errors.Wrap(err, "could not get post merge payload header")
 	}
-	return header.ParentHash(), nil
+	return header.BlockHash(), nil
```

Comment:
This function should be called `getBlockHash`, not `getParentBlockHash`, right? Same for Capella

Reply:
I also feel this is out of scope for this PR, changing the name of this function will make the proposer code less clear as the blockhash here is the parent block hash of the block we are proposing. Since the proposers are the only one calling these functions I think it's probably better to leave as is.